docs/OV_Runtime_UG/ov_infer_request.md

```diff
@@ -48,9 +48,9 @@ You can use `ov::InferRequest::infer`, which blocks the application execution, t
 
 @endsphinxtabset
 
-### Asynchronous mode
+### Asynchronous Mode
 
-Asynchronous mode can improve application's overall frame-rate, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. You can use `ov::InferRequest::start_async` to infer model in the asynchronous mode:
+Asynchronous mode can improve overall application frame-rate, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. You can use the `ov::InferRequest::start_async` to infer a model in the asynchronous mode:
```

Comment:
```suggestion
Asynchronous mode can improve overall application frame-rate. It is possible, because rather than wait for inference to complete, the app can keep working on the host, while the accelerator is busy. The `ov::InferRequest::start_async` can be used to infer a model in the asynchronous mode:
```